### PR TITLE
Feature/#33 get profiles endpoint added

### PR DESCRIPTION
**Pull request by ArtemKiriutin**
closes https://github.com/veraPDF/verapdf-webapp-server/issues/33

### CODE REVIEW

File: job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/entity/enums/Profile.java [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
-	PDFA_1_A("PDF/A-1A"),
-	PDFA_1_B("PDF/A-1B"),
-	PDFA_2_A("PDF/A-2A"),
-	PDFA_2_B("PDF/A-2B"),
-	PDFA_2_U("PDF/A-2U"),
-	PDFA_3_A("PDF/A-3A"),
-	PDFA_3_B("PDF/A-3B"),
-	PDFA_3_U("PDF/A-3U"),
-	PDFA_AUTO("PDF/A Auto-detect");
+	TAGGED_PDF("Tagged PDF", true),
```
Comment (BezrukovM) [inline job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/entity/enums/Profile.java:4]: lets not add here this flag. This enum uses on the workers side, while the flag is specific only to job server

File: job-service/server/src/main/java/org/verapdf/webapp/jobservice/server/controller/JobController.java [hunk 2]
```diff
@@ -29,4 +39,9 @@ public JobDTO getJob(@PathVariable UUID jobId) throws NotFoundException {
 		return jobService.getJobById(jobId);
 	}
 
+	@GetMapping("/profiles")
+	public List<ProfileDTO> getProfiles() {
```
Comment (BezrukovM) [inline job-service/server/src/main/java/org/verapdf/webapp/jobservice/server/controller/JobController.java:43]: this endpoint should be implemented in ProfilesController with `/api/profiles` path. The logic with available or not can be embedded into the controller (for now), ngix update required

File: job-service/server/src/main/java/org/verapdf/webapp/jobservice/server/controller/ProfileController.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	private List<ProfileDTO> profiles;
+
+	public ProfileController() {
+		this.profiles = Arrays.stream(Profile.values())
+		                      .map(profileName -> new ProfileDTO(profileName))
+		                      .filter(profileDTO -> profileDTO.isAvailable())
+		                      .collect(Collectors.toList());
+	}
+
+	@GetMapping()
```
Comment (BezrukovM) [inline job-service/server/src/main/java/org/verapdf/webapp/jobservice/server/controller/ProfileController.java:26]: remove unused `()`

File: job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/dto/ProfileDTO.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+import java.util.Objects;
+
+public class ProfileDTO {
+
+	private Profile profileName;
+	private String humanReadableName;
+	private boolean available;
+
+	public ProfileDTO(Profile profileName) {
```
Comment (BezrukovM) [inline job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/dto/ProfileDTO.java:13]: the constructor should has 2 arguments (Profile + boolean available). The available flag should be evaluated outside of dto, as dto should be simple

Comment (ArtemKiriutin) [reply job-service/model/src/main/java/org/verapdf/webapp/jobservice/model/dto/ProfileDTO.java]: The constructor now have 3 argument to let us define available and enabled profiles. (Profile, boolean available, boolean enabled)

**Pull request merged**